From 069efd194cf4cb225075440ae58137ea6375fb8b Mon Sep 17 00:00:00 2001
From: Hans Verkuil <hans.verkuil@cisco.com>
Date: Mon, 15 May 2017 15:13:00 +0200
Subject: [PATCH] configure.ac: clarify configure summary

Some of the texts are rather obscure and misleading. Fix those.

Signed-off-by: Hans Verkuil <hans.verkuil@cisco.com>
Signed-off-by: Hugues Fruchet <hugues.fruchet@st.com>
---
 configure.ac | 48 ++++++++++++++++++++++++------------------------
 1 file changed, 24 insertions(+), 24 deletions(-)

diff --git a/configure.ac b/configure.ac
index e468b3a..3c70969 100644
--- a/configure.ac
+++ b/configure.ac
@@ -475,28 +475,28 @@ cat <<EOF
 compile time options summary
 ============================
 
-    Host OS             : $host_os
-    X11                 : $x11_pkgconfig
-    GL                  : $gl_pkgconfig
-    glu                 : $glu_pkgconfig
-    libjpeg             : $have_jpeg
-    libudev             : $have_libudev
-    pthread             : $have_pthread
-    QT version          : $QT_VERSION
-    ALSA support        : $USE_ALSA
-
-    build dynamic libs  : $enable_shared
-    build static libs   : $enable_static
-
-    gconv               : $USE_GCONV
-
-    libv4l              : $USE_LIBV4L
-    v4l_plugins         : $USE_V4L_PLUGINS
-    v4l_wrappers        : $USE_V4L_WRAPPERS
-    libdvbv5            : $USE_LIBDVBV5
-    dvbv5-daemon        : $USE_DVBV5_REMOTE
-    v4lutils            : $USE_V4LUTILS
-    qv4l2               : $USE_QV4L2
-    v4l2-ctrl           : $USE_V4L2_CTL
-    v4l2-compliance     : $USE_V4L2_COMPLIANCE
+    Host OS                    : $host_os
+    X11                        : $x11_pkgconfig
+    GL                         : $gl_pkgconfig
+    glu                        : $glu_pkgconfig
+    libjpeg                    : $have_jpeg
+    libudev                    : $have_libudev
+    pthread                    : $have_pthread
+    QT version                 : $QT_VERSION
+    ALSA support               : $USE_ALSA
+
+    build dynamic libs         : $enable_shared
+    build static libs          : $enable_static
+
+    gconv                      : $USE_GCONV
+
+    dynamic libv4l             : $USE_LIBV4L
+    v4l_plugins                : $USE_V4L_PLUGINS
+    v4l_wrappers               : $USE_V4L_WRAPPERS
+    libdvbv5                   : $USE_LIBDVBV5
+    dvbv5-daemon               : $USE_DVBV5_REMOTE
+    v4lutils                   : $USE_V4LUTILS
+    qv4l2                      : $USE_QV4L2
+    v4l2-ctl uses libv4l       : $USE_V4L2_CTL
+    v4l2-compliance uses libv4l: $USE_V4L2_COMPLIANCE
 EOF
-- 
1.9.1

